codebase2/codebase-sqlite/U/Codebase/Sqlite/Patch/TermEdit.hs

```diff
@@ -22,6 +22,12 @@ type Referent' t h = Referent.Referent' (Reference' t h) (Reference' t h)
 data TermEdit' t h = Replace (Referent' t h) Typing | Deprecate
   deriving (Eq, Ord, Show)
 
+instance Functor (TermEdit' t) where
```

Comment:
interestingly the default deriving doesn't work:

```
U/Codebase/Sqlite/Patch/TermEdit.hs:23:17: error: [GHC-16437]
    • Can't make a derived instance of ‘Functor (TermEdit' t)’:
        Constructor ‘Replace’ must use the type variable only as the last argument of a data type
    • In the data declaration for ‘TermEdit'’
   |
23 |   deriving (Eq, Functor, Ord, Show)
   |                 ^^^^^^^
```

Reply:
Not sure, but I'd guess the default `Functor` can only figure out one type variable?